cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -44,7 +44,9 @@ class CxTowerPlanLine(models.Model):
         help="Actions trigger based on command result."
         " If empty next command will be executed",
     )
-    command_code = fields.Text(related="command_id.code", readonly=True)
+    command_code = fields.Text(
```

Comment:
This approach doesn't allow to check those lines if required.
So we need to add a compute o2m field for lines and hide/show it based on the command action.